example/src/minimal_publisher.cpp

```diff
@@ -26,61 +26,48 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-#include "admittance_controller_parameters.hpp"
-#include "rclcpp/rclcpp.hpp"
+#include <rclcpp/rclcpp.hpp>
+
+#include <admittance_controller_parameters.hpp>
+#include <generate_parameter_library_example/minimal_publisher.hpp>
```

Comment:
Usually, first include is the .hpp file with the same name. This is a strategy what I am always using, because this file is the most relevant.

This would be the order we are usually using in ros2_control

```suggestion
#include <generate_parameter_library_example/minimal_publisher.hpp>

#include <admittance_controller_parameters.hpp>
#include <rclcpp/rclcpp.hpp>
```

This could be another way, nevertheless, I would always put the direct `.hpp` file first. This is what some linters also recommend.

```suggestion
#include <generate_parameter_library_example/minimal_publisher.hpp>

#include <rclcpp/rclcpp.hpp>

#include <admittance_controller_parameters.hpp>
```

Reply:
These changes are now part of the latest version of this PR. The order of the includes is set by clang-format.

Reply:
Done